LICENSE

```diff
@@ -1,27 +0,0 @@
-Copyright (c) 2015-2017, conda-forge
-All rights reserved.
```

Comment:
FYI this is the license I was referring to as having been moved. We do want to make sure to keep this one as it is the license for staged-recipes as a whole. Made sure it is kept in the rebased PR.